### DISCUSSION TITLE
Removing ehcache dependency as it has vulnerabilities

### DISCUSSION THREAD

**Discussion by JurisPipurs** [playframework/playframework#11570] [Q&A]
Hello, 
The latest version of net.sf.ehcache contains vulnerabilities and I would like to remove it from my sbt project and use caffeine instead.

If I add 
excludeDependencies ++= Seq(ExclusionRule("net.sf.ehcache", "ehcache"))
to my build.sbt file, I get NoClassDefFoundError: net/sf/ehcache/CacheManager when trying to run the project.

My question is: is it possible to remove ehcache dependency from the project?

I'm currently using play framework version 2.8.15

**Comment by tsuyoshizawa** [comment]
Can you find the following dependency in your project?

```
libraryDependencies ++= Seq(
  ehcache
)
```

https://www.playframework.com/documentation/2.8.x/ScalaCache#EhCache

Play Framework does not include ehcache by default.
